student-work/2023/Sharon-Schnee/api-final-project/readme.md

```diff
@@ -1 +1,78 @@
+# Hamburger POC API Guide
+
```

Comment:
## Additional Information

Reply:
@Schnee18 - consider moving this to the end of the page

Reply:
Thanks for the feedback! Two questions:

1. What should I move to the end of the page? The heading "Hamburger POC API Guide"? 
2. Also for the links to different parts of the readme, I wanted it in a list. Did you just want the numbers taken out, or should it not be in a list?

Reply:
the links to additional information should be moved to the end of the page